modules/candidate_parameters/test/TestPlan.md

```diff
@@ -69,51 +69,59 @@
         * Make sure they update properly in the front-end and backend
 
 ### Date of Birth Tab
-33. Check that date of birth can only be amended if user has candidate_dob_edit permission.
-34. Change the date of birth and try to save it. Check that it is updated on front-end and database.
-35. Check that entering a date later than today gives a data entry error.
-36. Check that the date stored matches the _DOB Format_ specified in the configuration module. 
+1. Check that date of birth can only be amended if user has candidate_dob_edit permission.
+2. Change the date of birth and try to save it. Check that it is updated on front-end and database.
+3. Check that entering a date later than today gives a data entry error.
+4. Check that the date stored matches the _DOB Format_ specified in the configuration module. 
 
 ### Date of Death Tab
-37. Check that date of death can only be set if user has candidate_dod_edit permission.
-38. Change the date of death and try to save it. Check that it is updated on front-end and database.
-39. Check that entering a date of death before the date of birth gives a data entry error.
-40. Check that entering a date later than today gives a data entry error.
-41. Check that the date stored matches the _DOD Format_ specified in the configuration module.
+1. Check that date of death can only be set if user has candidate_dod_edit permission.
+2. Change the date of death and try to save it. Check that it is updated on front-end and database.
+3. Check that entering a date of death before the date of birth gives a data entry error.
+4. Check that entering a date later than today gives a data entry error.
+5. Check that the date stored matches the _DOD Format_ specified in the configuration module.
+
+### Diagnosis Evolution Tab
+1. Confirm that all the fields in this panel correspond to what's stored in the candidate table.
+	* PSCID
+    * DCCID
+    * Latest Diagnosis
```

Comment:
```suggestion
    * PSCID
    * DCCID
    * Latest Diagnosis
```

Reply:
for some reason, a project diagnosis evolution configuration is being applied to a candidate's visit in another project
<img width="1206" alt="Screen Shot 2022-12-19 at 1 50 03 PM" src="https://user-images.githubusercontent.com/17415878/208498501-cf3a7503-cafd-4f99-b897-661f933bf8e2.png">
<img width="1209" alt="Screen Shot 2022-12-19 at 1 50 15 PM" src="https://user-images.githubusercontent.com/17415878/208498524-832a7d6b-c26b-4188-8ec2-5a06b741bd83.png">